packages/angular/cli/lib/config/schema.json

```diff
@@ -623,6 +623,10 @@
               "type": "string",
               "description": "Path where output will be placed."
             },
+            "assetsOutputPath": {
```

Comment:
`resources` instead of `assets`.  Assets already have a distinct meaning and behavior.